src/Workspaces/Remote/ServiceHub/Services/NavigateToSearch/RemoteNavigateToSearchService.cs

```diff
@@ -37,6 +37,18 @@ public RemoteNavigateToSearchService(in ServiceConstructionArguments arguments,
                 cancellationToken).ConfigureAwait(false);
         }
 
+        public ValueTask HydrateAsync(PinnedSolutionInfo solutionInfo, CancellationToken cancellationToken)
```

Comment:
so the navto service is Project affinitized.  I don't really want N project calls calling out to OOP to hydrate (since hydration is at the SOlution granularity for OOP).  I started there, but moved to this model to more closely match the underlying semantics we can actually provide.